dali/aux/optical_flow/turing_of/optical_flow_turing.cu

```diff
@@ -50,6 +50,18 @@ pitch_xy(T *buffer, ptrdiff_t x, ptrdiff_t y, ptrdiff_t pitch_bytes) {
   return reinterpret_cast<T *>(reinterpret_cast<uintptr_t>(buffer) + pitch_bytes * y)[x];
 }
 
+
+template<typename ConversionMethod>
+void ColorConversion(ConversionMethod cvtm, const uint8_t *input, uint8_t *output, size_t pitch,
```

Comment:
I am not sure if `ColorConversion` the right name for this function. You are actually doing 2 things, copying to the right layout and converting the color space to RGB. The names got me slightly confused about whats happening.

Maybe `ConvertToOFLayout` would be more suited.

And `ConversionMethod` to `ColorConversionMethod`

Reply:
👍